reactNativeProject/src/screens/Main/components/Item/styles.js

```diff
@@ -1,6 +1,12 @@
 import { StyleSheet } from "react-native";
 import colors from "../../../../constants/colors";
-import variables from "../../../../constants/variables";
+
+const variables = {
+  imageRadio: 32,
```

Comment:
Why is `imageRadio` different than `circleWidth`? One is probably slightly large than the other, and they should be equal.

Use a single constant for both styles